static/css/cdr-homepage.css

```diff
@@ -1,262 +1,209 @@
 @import url("https://fonts.googleapis.com/css?family=Libre+Baskerville:400i|Open+Sans:400,600");
-/* line 17, ../sass/dcr.scss */
-* {
-    font-family: 'Open Sans', sans-serif;
-}
+body {
```

Comment:
Why are all these formatting changes suddenly coming through?

Reply:
I added in the font and style updates in this pull request https://jira.lib.unc.edu/browse/BXC-2368